<use f='qv2ray/test/catch.hpp' l='2971' c='_ZNK5Catch12IRegistryHub27getStartupExceptionRegistryEv'/>
<def f='qv2ray/test/catch.hpp' l='12391' ll='12399'/>
<use f='qv2ray/test/catch.hpp' l='12460' c='_ZNK5Catch12_GLOBAL__N_111RegistryHub27getStartupExceptionRegistryEv'/>
<use f='qv2ray/test/catch.hpp' l='12496'/>
<size>24</size>
<fun r='_ZN5Catch24StartupExceptionRegistry3addERKNSt15__exception_ptr13exception_ptrE'/>
<fun r='_ZNK5Catch24StartupExceptionRegistry13getExceptionsEv'/>
<mbr r='Catch::StartupExceptionRegistry::m_exceptions' o='0' t='std::vector&lt;std::exception_ptr&gt;'/>
<fun r='_ZN5Catch24StartupExceptionRegistry3addERKNSt15__exception_ptr13exception_ptrE'/>
<fun r='_ZNK5Catch24StartupExceptionRegistry13getExceptionsEv'/>
